src/pages/ReportDetailsPage.tsx

```diff
@@ -328,6 +360,25 @@ function ReportDetailsPage({policies, report, session, personalDetails}: ReportD
                             )}
                         </View>
                     </View>
+                    {shouldShowRoomName && (
+                        <View style={[styles.flex1, !shouldDisableRename && styles.mt3, isGroupChat && styles.mb5]}>
```

Comment:
This should be wrapped in `OfflineWithFeedback`, as it was on `ReportSettingsPage`